src/config.ts

```diff
@@ -108,28 +113,30 @@ async function fetchProvider(source: ProviderSource): Promise<HttpProvider | gan
   }
 }
 
-async function fetchAccount(source: AccountSource, web3: Web3): Promise<string | undefined> {
+async function fetchAccount(source: AccountSource, web3: Web3): Promise<AccountConfig | undefined> {
   if (!source) {
     return undefined;
   } else if ('unlocked' in source) {
     // We'll actually ping the provider 😬
     let accounts = await web3.eth.getAccounts();
     let index = Number(source.unlocked);
-
-    return accounts[index];
+    return {default_account: accounts[index], wallet_accounts: []};
   } else if ('env' in source) {
     let privateKey = process.env[source.env];
     if ( privateKey )   {
       let account = web3.eth.accounts.wallet.add(privateKey);
-      return account.address;
+      return {default_account: account.address, wallet_accounts: []};
     } else {
       return undefined;
     }
   } else if ('file' in source) {
     try {
-      let privateKey = await readFile(source.file, 'utf8');
-      let account = web3.eth.accounts.wallet.add('0x' + privateKey.trim());
-      return account.address;
+      let privateKeys = (await readFile(source.file, 'utf8')).split('\n');
+      const wallet_accounts = privateKeys.filter(key => !!key.trim()).map((key) => {
+        const wallet = web3.eth.accounts.wallet.add('0x' + key);
+        return wallet.address;
+      });
+      return {default_account: wallet_accounts[0], wallet_accounts: wallet_accounts};
```

Comment:
any problem with default account appearing in wallet_accounts too?

Reply:
dont think so? the purpose of `default_account` is to set the default `from` in web3, and to expose an `account` attribute on the saddle obj, to be more similar to truffle